sdk/eventhub/Microsoft.Azure.EventHubs/tests/Processor/ProcessorNegativeCases.cs

```diff
@@ -14,60 +14,70 @@ namespace Microsoft.Azure.EventHubs.Tests.Processor
 
     public class NegativeCases : ProcessorTestBase
     {
+        //Update Live test to use EventHub managed by EventHubScope
```

Comment:
I don't see these comments as adding value for future maintainers;  please remove.